server/src/main/java/org/elasticsearch/index/mapper/FieldMapper.java

```diff
@@ -1466,6 +1466,14 @@ public static BiConsumer<String, MappingParserContext> notInMultiFields(String t
         };
     }
 
+    public static BiConsumer<String, MappingParserContext> notDynamicallyCreated(String type) {
```

Comment:
The issue isn't that its dynamically created, but that its dynamically created via a template. Could you adjust this field name?